### DISCUSSION TITLE
How do you stop cmp from adding texts in comments to completion list?

### DISCUSSION THREAD

**Discussion by rudfol** [hrsh7th/nvim-cmp#1521] [Q&A] [answered]
Hi there,

Many times there are words from comments that clutter up the completion list. Would love to know how to disable them please.

In prestige of your work,

Joon

**Comment by fitrh** [comment, marked as answer]
The easiest way is to not use the `buffer` source and then skip the `Text` kind of LSP source with

```lua
  {
    name = 'nvim_lsp',
    entry_filter = function(entry, ctx)
      return require('cmp.types').lsp.CompletionItemKind[entry:get_kind()] ~= 'Text'
    end
  }
```

**Comment by rudfol** [comment]
fking hell thank you. It was resolved so easily. I'm so happy, kudos
